.editorconfig

```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
So, this is where it gets tricky. We use 4-space indents for .py and .js files. For .less files we use 2-space indents. I don't know offhand whether there are other rules, but I think we should move the indent bits to a `[*.py,*.js]` block and add this:

```
[*.less]
indent_style = space
indent_size = 2
```